### DISCUSSION TITLE
Automate generation of oxlint config

### DISCUSSION THREAD

**Discussion by nicksrandall** [oxc-project/oxc#4279] [Feature Request]
It would be nice if there was a command (or docs) to create an up-to-date `.oxlintrc.json` file with all of the latest recommended rules. It's currently a pain to migrate from CLI to config file and maintain feature parity.

**Comment by nicksrandall** [comment]
It would be better if there was a command that allowed you to pass same arguments as cli (to compose rules) and then it would print out a config file with all enabled rules turned on.

**Reply by mysteryven** [reply]
I think we can add [`print-config`](https://eslint.org/docs/latest/use/command-line-interface#--print-config).
